web/src/app/modules/site/components/navTopBar.jsx

```diff
@@ -74,7 +74,7 @@ const LocationIndicator = ({ provider, location }) => {
       providerLabel = `AWS${location}`;
       break;
     case ProviderEnum.ONPREM:
-      providerLabel = 'On premise';
+      providerLabel = 'On premises';
```

Comment:
@alex-kovoy Just making sure - this will not break anything, will it? I.e. we don't search anything anywhere in the UI by this label?

Reply:
looks good.